### PR TITLE
Feature- sidebar navigation

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**  
_Describe what the pull-request is about_

- [x] New design for left navigation
- [x] Mobile navigation
- [x] Added toggle functionality
- [x] Using variables
- [x] Colors
- [x] Spacing 
- [x] Boostrap toggle removed 
- [x] Caret icon added (scania icon, until new icons)
- [x] Add mobile toggle (hamburger menu, logic)
- [x] Close menu on menu click (Logic)

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #113 

**How to test**  
_Add description how to test if possible_

- [x] Design from figma
- [x] Run navigation in desktop/mobile **Need to add class`sdds-nav-open` on the `<nav>` element**
- [x] Run in different browsers
- [x] Run in different breakpoints
- [x] Toggle menu
- [x] Switch around in navigation

**Additional context**  
_Add any other context about the pull-request here._
- Needs header before toggle function
- Some sites are broken, needs the latest page detail before merge

### CODE REVIEW

**Comment by niklasros** [comment]
The container behind the detail page when active has the wrong colour. Should be $grey-800.

side-nav-detail-active

**Comment by niklasros** [comment]
Hover state for the detail page links are missing.

**Comment by niklasros** [comment]
The left-padding of the links should be 16px and the right-padding of the icon-arrow should be 16px. Now it is 24.

nav-expanded-mobile

**Comment by niklasros** [comment]
The margin between the main links and the external links could be smaller on mobile breakpoint than desktop. In Figma, we used 64px for mobile.

**Comment by niklasros** [comment]
Add rotation animation for the icon-arrow when the link is being clicked.

**Comment by niklasros** [comment]
The horizontal dividers on header and side-nav is still misaligned.

header-sidenav-dividers

**Comment by niklasros** [comment]
The symbol within the header should lead back to the startpage when clicked. Currently nothing happens.

**Comment by helloimela** [comment]
image
Padding on a link hover should be 12px.
Maybe start using our spacing variables for 'a' element? so padding: var(--sdds-element-spacing-12) var(--sdds-element-spacing-16)

File: src/app/components/page/tab-content/tab-content.component.ts [hunk 1]
```diff
@@ -53,13 +53,11 @@ export class TabContentComponent {
         }
       }
 
-      console.log(this.tabContent)
```
Comment (helloimela) [inline src/app/components/page/tab-content/tab-content.component.ts:56]: Remove Console.log()

**Review by helloimela — APPROVED** [review]
Great job!

**Pull request merged**